h2o-core/src/main/java/water/H2ONode.java

```diff
@@ -53,12 +53,12 @@ public void stopSendThread(){
   /** Identification of the node via IP and PORT.
    *
    */
-  static final class H2Okey extends InetSocketAddress implements Comparable {
+  public static final class H2Okey extends InetSocketAddress implements Comparable {
```

Comment:
Or I don't need to use class `H2OKey` at all and I could list through all available nodes in `LogsHandler` until I find the one with correct IP. This way there would be no changew to the `_memary` and `H2OKey`, but for the price of listing the nodes. What do you think @mmalohlava  ?